django/core/management/commands/makemigrations.py

```diff
@@ -98,7 +99,13 @@ def handle(self, *app_labels, **options):
                     for app_label in consistency_check_labels
                     for model in apps.get_app_config(app_label).get_models()
             )):
-                loader.check_consistent_history(connection)
+                try:
+                    loader.check_consistent_history(connection)
+                except OperationalError as error:
+                    raise CommandError(
```

Comment:
Can we log a _warning_ here rather than raise a `CommandError`? That way you are able to run this command without a database connection at all.

Right now you would be hiding the ugly traceback, which is good, but still preventing someone from running this if they don't have a connection.

Reply:
+1 to some form of runtime warning.